src/renderer/services/wallet/types.ts

```diff
@@ -36,13 +38,25 @@ export type KeystoreService = {
   validatePassword$: (password: string) => LiveData<Error, null>
 }
 
+/**
+ * Wraps WalletBalancesRD into an object to provide extra information (`Address` + `Chain` + `WalletType`)
+ * Currently needed in `AssetView` - TODO(@Veado) Think about to extract it into view layer (as helper or so)
```

Comment:
seems this is semantically correct way to store this data together as every operation is also depends on `WalletType`. is not it ?

Reply:
Not sure if I understand your comment ^ correctly: `WalletBalance` (as part of `balances: WalletBalancesRD` in `ChainBalances`) already includes information about `Address` + `Chain` + `WalletType`. With that we have same data in `ChanBalances` - which is a duplication of data. However, as mentioned in the comment we use `ChainBalances`  in `AssetView` to build a nested table of balances. Each nested table has a header, which displays an `Address` and in near future it will show a wallet type as well. To show these data instantly in the header without the need to load balances, we have to put it into `ChainBalance` as extra properties besides `balances.`